server/protocols/webadmin-cli/README.md

```diff
@@ -32,7 +32,7 @@ $ ./james-cli --url http://127.0.0.1:9999 domain list
 The above command lists all domain names available on domain route at address http://127.0.0.1:9999. 
 It does not require any argument to execute. 
 
-#### Options are optional:
+Options are optional:
```

Comment:
I suggest `The following options can be used:`

as options are necessarily optionals, overwise we would not call them options...